src/promise-katas-1.js

```diff
@@ -33,71 +33,210 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 // when we call returnPromise like 'returnPromise()' we expect our function to use the
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => { 
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve();
+    }, 1000);
+  });
+};
 
 // 2 Create a function that returns a promise which returns 10 if it resolves succesfully
 // Example:
 // returnTen() resolves to the value 10
 
-const returnTen = () => { };
+function returnTen() {
+  return new Promise((resolve) => {
+    resolve(10);
+  });
+}
+  
+  returnTen().then(value => {
+    console.log(value)
+  }).catch(error => {
+    console.error('The promise is expected to resolve with 10!');
+  });
 
 // 3 Create a function that returns a promise which returns 'string' if it resolves succesfully
 // Example:
 // returnString() resolves to the value 'string'
 
-const returnString = () => { };
+function returnString() { 
+  return new Promise((resolve) => {
+    resolve('string')
+  });
+}
+
+returnString().then(value => {
+  console.log(value)
+}).catch(error => {
+  console.error("The promise is expected to resolve with 'string'!")
+});
 
 // 4 Create a function that returns a promise which returns an object with a key of name and value of 'Bob' if it resolves succesfully
 // Example:
 // returnBob() resolves to the value { name: 'Bob' }
 
-const returnBob = () => { };
+const returnBob = (name) => {
+  return new Promise((resolve) => {
+      const object = { name: 'Bob' };
+      resolve(object);   
+    });
+}
+
+returnBob({ name: 'Bob' })
+.then(object => {
+  console.log(object);
+}).catch(error => {
+  console.error("The promise is expected to resolve with { name: 'Bob' }!");
+});
 
 // 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
 // Example:
 // returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
 
-const returnList = () => { };
+const returnList = () => {
+  return new Promise((resolve) => {
+    const array = ['eggs', 'apples', 'milk', 'bread'];
+    resolve(array);
+  });
+}
+
+returnList(['eggs', 'apples', 'milk', 'bread'])
+.then(array => {
+  console.log(array);
+}).catch(error => {
+  console.error("The promise is expected to resolve with ['eggs', 'apples', 'milk', 'bread']!");
+});
 
 // 6 Create a function that returns a promise which returns "An error occurred" if it rejects
 // Example:
 // anError() rejects to "An error occurred"
 
-const anError = () => { };
+const anError = () => {
+  return new Promise((resolve, reject) => {
```

Comment:
Could be refactored as:
````
const anError = () => new Promise((_, reject) => reject("An error occurred"));
````
You can swap resolve to a _ when there is going to be reject returned only. The first parameter is always required, but can be swapped to _ when it is not used